testsupport/wait/host.go

```diff
@@ -415,6 +416,22 @@ func UntilMasterUserRecordHasTierName(expected string) MasterUserRecordWaitCrite
 	}
 }
 
+func UntilMasterUserRecordHasNoTierHashLabel() MasterUserRecordWaitCriterion {
+	return MasterUserRecordWaitCriterion{
+		Match: func(actual *toolchainv1alpha1.MasterUserRecord) bool {
+			if actual.Labels == nil {
```

Comment:
I believe this is not needed, `range` will not fail if `actual.Labels` is `nil`. (but it's a minor thing, not needed to be addressed in this PR)

Reply:
You're right, good catch. Done in d279172377307fd298f562332a8e34f5b5910b0d